ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -310,12 +312,12 @@ SettingsContentBase {
             Layout.fillWidth: true
 
             tokenListUpdatedAt: tokensStore.tokenListUpdatedAt
+            assetsController: root.assetsStore.assetsController
+            collectiblesController: root.collectiblesStore.collectiblesController
             sourcesOfTokensModel: tokensStore.sourcesOfTokensModel
             tokensListModel: tokensStore.extendedFlatTokensModel
-            baseWalletAssetsModel: RootStore.walletAssetsStore.groupedAccountAssetsModel
-            baseWalletCollectiblesModel: {
-                return RootStore.collectiblesStore.allCollectiblesModel
-            }
+            baseWalletAssetsModel: root.assetsStore.groupedAccountAssetsModel
```

Comment:
Why do we need them here? Models should be the same than `XXcontroller.sourceModel` so I think they can be removed!

Reply:
I think it's more flexible to pass them separately, but ymmv

Reply:
But what will happen if we pass different models (I mean different than the one controlled by the controller)??? Then we can have inconsistencies. I don't see it as a flexibility but as a potential cause of issue.

I think we need to remove them.

Reply:
OK, I'll try this out

Reply:
Fixed